APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1141,6 +1208,29 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 }
             }
                 break;
+                
+            case kAPCUserInfoItemTypeDateOfBirth:
+            {
+                if (self.canEditBirthDate &&
+                    ![self isEditingAllowedForHealthKitProperty:HKCharacteristicTypeIdentifierDateOfBirth])
+                {
+                    [self showEditingNotAllowedAlertForHealthKitProperty:HKCharacteristicTypeIdentifierDateOfBirth];
+                    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+                }
+                else
+                {
+                    APCTableViewItem *field = [self itemForIndexPath:indexPath];
+                    if (self.isEditing && field.isEditable) {
+                        [super tableView:tableView didSelectRowAtIndexPath:indexPath];
+                    }
+                }
+            }
+                break;
+                
+            case kAPCUserInfoItemTypeHeight:
+            {
+                [self setEditing:!self.isEditing];
```

Comment:
wait, won't this just flip on or off editing for the whole table view controller every time this method gets called if there's an item of this type?